Sources/OpenAPIRuntime/Errors/RuntimeError.swift

```diff
@@ -141,3 +141,43 @@ internal enum RuntimeError: Error, CustomStringConvertible, LocalizedError, Pret
 @_spi(Generated) public func throwUnexpectedResponseBody(expectedContent: String, body: any Sendable) throws -> Never {
     throw RuntimeError.unexpectedResponseBody(expectedContent: expectedContent, body: body)
 }
+
+/// HTTP Response status definition for ``RuntimeError``. 
+extension RuntimeError: HTTPResponseConvertible {
+    public var httpStatus: HTTPTypes.HTTPResponse.Status {
+        switch self {
+        case .invalidServerURL,
+                .invalidServerVariableValue:
+            .notFound
+        case .invalidExpectedContentType,
+                .missingCoderForCustomContentType,
+                .unexpectedContentTypeHeader:
+            .unsupportedMediaType
+        case .unexpectedAcceptHeader(_):
+            .notAcceptable
+        case .missingOrMalformedContentDispositionName:
+            .unprocessableContent
```

Comment:
> The HTTP 422 Unprocessable Content [client error response](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses) status code indicates that the server understood the content type of the request content, and the syntax of the request content was correct, but it was unable to process the contained instructions.
— https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422


Sounds like this one might not be the correct response for missing or malformed. Should we return 401 instead here?

Reply:
Or a generic 400 is also fine, if it's a malformed header